app/js/controller/list_controller.js

```diff
@@ -137,9 +145,22 @@ export default class ListController extends Controller {
     }
 
     installReq.onerror = (err) => {
-      this.appView.showAlertDialog('Error installing: ' + err.target.error.name);
-      console.log('install error', err);
+      var errorType = err.target.error.name;
+      switch(errorType) {
+        case 'DENIED':
+          // If the user cancelled the install, we do nothing.
+          break;
+
+        case 'NETWORK_ERROR':
+          this.showAlertDialog('Install Error: No network');
+          break;
+
+        default:
+          this.showAlertDialog('Error installing: ' + err.target.error.name);
+          break;
+      }
```

Comment:
I'll leave it as is for now, but will consider refactoring in the future if this starts to grow unwieldy.